src/client/features/search/landing-box.js

```diff
@@ -122,37 +124,59 @@ synonyms:"TP53, BCC7, LFS1, P53, TRP53"
 }]
 */
 const getLandingResult= (query)=> {
-  const q=query.trim();
+  let q=query.trim().split(' ');
+  let ncbiIds={},uniprotIds={},labeledId={};
+  const genesToSearch = [];
+  q.forEach ((id)=> {
+    const splitId=id.split(':');
+
+    //Parse the query based on the format
+    if(/uniprot:\w+$/.test(id)) {
```

Comment:
Make case insensitive

Reply:
i.e. the prefix part (id is case-sensitive)